src/platforms/spark/ApplicationTexture.mjs

```diff
@@ -1,6 +1,22 @@
 import lng from "wpe-lightning/src/lightning.mjs";
 
-export default class ApplicationTexture extends lng.Texture {
+var appTextures = [];
+
+function clearAppTexture(texture) {
```

Comment:
Does the app not destroy on GC, why doing this explicitly?

Reply:
Seen a reference to sparkwebgl is there inside created context inside initGL/ESMLoader and causing leak. Tried clearing out in initGL.js , asfter which issue is not seen and seen all the App and WebGLTexture objects etc got destroyed from inspector.